examples/local_actions_example/main.py

```
@@ -0,0 +1,211 @@
+    expecting the server to reject them. This is the payoff of the auth
+    feature — proof that admin port reachability alone no longer lets a
+    caller act as the operator."""
+    wrong_key = secrets.token_hex(32)
+    print("\n[wrong-key check] signing with a freshly generated key (NOT the operator)")
+    wrong_local = LocalClient(ADMIN_URL, private_key=wrong_key)
+    try:
+        wrong_local.create_stream(
+            generate_stream_id("wrong-key-attempt"),
+            STREAM_TYPE_PRIMITIVE,
+        )
+    except Exception as rejected:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)